contracts/contracts/Poll.sol

```diff
@@ -171,6 +192,9 @@ contract Poll is Params, Utilities, SnarkCommon, EmptyBallotRoots, IPoll {
     batchHashes.push(NOTHING_UP_MY_SLEEVE);
     updateChainHash(placeholderLeaf);
 
+    InternalLazyIMT._init(pollStateTree, extContracts.maci.stateTreeDepth());
```

Comment:
I think each poll should have configurable state tree depth, some might want to go smaller than the maci one